### PR TITLE
Add release notes for 10.0.10

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: docs/src/main/paradox/scala/http/release-notes.md [hunk 1]
```diff
@@ -5,7 +5,25 @@
...
 
 It is now possible to connect to @scala[@ref[HTTP(S) Proxies](client-side/client-transport.md)]@java[@ref[HTTP(S) Proxies](client-side/client-transport.md)]
-that require an authorization via an `Proxy-Authorization` header. This can be set up directly on the `ClientTransport` object when configuring the proxy.
+that require an authorization via an `Proxy-Authorization` header. This can be set up directly on the `ClientTransport` object when configuring the proxy. ([#1213](https://github.com/akka/akka-http/issues/1213))
+
+### Improvements
+
+* (server) Better error reporting when failing to handle CONNECT requests ([#1315](https://github.com/akka/akka-http/issues/1315))
+* Add HttpApp.startServer(host, port, system) ([#1294](https://github.com/akka/akka-http/issues/1294))
+* Logging with println in Http2ServerDemux ([#1275](https://github.com/akka/akka-http/issues/1275))
```
Comment (jrudolph) [inline docs/src/main/paradox/scala/http/release-notes.md]: I guess that was rather a bugfix. Let's make sure that all lines are phrased in the same way, i.e. not stating the bug but the fix. In this case e.g. it's "Remove println logging in Http2 implementation".

File: docs/src/main/paradox/scala/http/release-notes.md [hunk 2]
```diff
@@ -5,7 +5,25 @@
 ### Support for HTTP(S) proxies with Authorization
```
Comment (jrudolph) [inline docs/src/main/paradox/scala/http/release-notes.md]: We should add another point that we now have documentation for the experimental Http2 support and link to it (and if it's not yet merge, let's finish it before the release).

File: docs/src/main/paradox/scala/http/release-notes.md [hunk 3]
```diff
@@ -5,7 +5,25 @@
...
+* Http2: Inbound flow control ([#737](https://github.com/akka/akka-http/issues/737))
+
+### Bugfixes
+
+* HttpChallenge rendering is broken if realm parameter is None ([#1295](https://github.com/akka/akka-http/issues/1295))
+* Incorrect request URIs with HTTP/2 ([#1274](https://github.com/akka/akka-http/issues/1274)
+* HttpResponseParser returns invalid status code for statuses without reason phrase ([#1251](https://github.com/akka/akka-http/issues/1251))
+* requests hang if pool is shutdown before receiving response ([#1245](https://github.com/akka/akka-http/issues/1245))
+* Large response entity is truncated over https with 'Connection: close' header ([#1199](https://github.com/akka/akka-http/issues/1199))
+
```
Comment (jrudolph) [inline docs/src/main/paradox/scala/http/release-notes.md]: #1219 (also known as face38e) seems to be missing.

File: docs/src/main/paradox/scala/http/release-notes.md [hunk 4]
```diff
@@ -5,7 +5,25 @@
...
+that require an authorization via an `Proxy-Authorization` header. This can be set up directly on the `ClientTransport` object when configuring the proxy. ([#1213](https://github.com/akka/akka-http/issues/1213))
+
+### Improvements
+
+* (server) Better error reporting when failing to handle CONNECT requests ([#1315](https://github.com/akka/akka-http/issues/1315))
+* Add HttpApp.startServer(host, port, system) ([#1294](https://github.com/akka/akka-http/issues/1294))
+* Logging with println in Http2ServerDemux ([#1275](https://github.com/akka/akka-http/issues/1275))
+* Missing final boundary for empty multipart entities ([#1257](https://github.com/akka/akka-http/issues/1257))
+* Add Cache-Control 'immutable' value ([#1212](https://github.com/akka/akka-http/issues/1212))
+* Http2: Inbound flow control ([#737](https://github.com/akka/akka-http/issues/737))
```
Comment (jrudolph) [inline docs/src/main/paradox/scala/http/release-notes.md]: #1270 is missing.

File: docs/src/main/paradox/scala/http/release-notes.md [hunk 5]
```diff
@@ -5,7 +5,25 @@
...
+
+### Bugfixes
+
+* HttpChallenge rendering is broken if realm parameter is None ([#1295](https://github.com/akka/akka-http/issues/1295))
+* Incorrect request URIs with HTTP/2 ([#1274](https://github.com/akka/akka-http/issues/1274)
+* HttpResponseParser returns invalid status code for statuses without reason phrase ([#1251](https://github.com/akka/akka-http/issues/1251))
+* requests hang if pool is shutdown before receiving response ([#1245](https://github.com/akka/akka-http/issues/1245))
```
Comment (jrudolph) [inline docs/src/main/paradox/scala/http/release-notes.md]: "Ensure that no responses are lost when `host-connection-pool.idle-timeout` kicks in"

File: docs/src/main/paradox/scala/http/release-notes.md (hunk 3 above)
Comment (jrudolph) [reply docs/src/main/paradox/scala/http/release-notes.md]: Ah, it seems that would be the above line for which the issue id it just wrong. So, changing #1199 -> #1219 should fix it.

File: docs/src/main/paradox/scala/http/release-notes.md (hunk 2 above)
Comment (ktoso) [reply docs/src/main/paradox/scala/http/release-notes.md]: :+1:

File: docs/src/main/paradox/scala/http/release-notes.md [hunk 6]
```diff
@@ -5,7 +5,40 @@
...
 It is now possible to connect to @scala[@ref[HTTP(S) Proxies](client-side/client-transport.md)]@java[@ref[HTTP(S) Proxies](client-side/client-transport.md)]
-that require an authorization via an `Proxy-Authorization` header. This can be set up directly on the `ClientTransport` object when configuring the proxy.
+that require an authorization via an `Proxy-Authorization` header. This can be set up directly on the `ClientTransport` object when configuring the proxy. ([#1213](https://github.com/akka/akka-http/issues/1213))
+
+### Documentation for HTTP 2 support (Preview)
+
+Server-side HTTP/2 support, currently available as a preview, has now been
+@ref[included in the documentation](server-side/http2.md)
+([#1297](https://github.com/akka/akka-http/pull/1297)
+
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/release-notes.md]: Great, will likely get us more people trying it... Brace yourself...

File: docs/src/main/paradox/scala/http/release-notes.md [hunk 7]
```diff
@@ -5,7 +5,40 @@
...
+### Documentation consolidation
+
+Behind the scenes, a lot of work has been done on the ongoing effort to
+consolidate the Java and Scala versions of the documentation, notably
+by
+Jonas Fonseca and Josep Prat. This will make our documentation more consistent,
+higher-quality and more easy to browse and maintain ([#1290](https://github.com/akka/akka-http/issues/1290)
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/release-notes.md]: Yup, great 👍

**Pull request merged**